src/commands/Admin/eval.ts

```diff
@@ -15,20 +9,15 @@ const command: Command = {
   usage: "+eval <code>",
   allowTip: false,
   allowDM: true,
+  roleWhitelist: ["620274909700161556"], //*
+  channelWhitelist: ["624881429834366986", "730042612647723058", "551693960913879071"], // staff-bots bot-development admin-bots
   options: [{
     type: "STRING",
     name: "code",
     description: "The code to run",
     required: false
   }],
-  async execute(interaction: Discord.CommandInteraction, getString: (path: string, variables?: { [key: string]: string | number } | string, cmd?: string, lang?: string) => any) {
-    const me = interaction.member,
-      guild = interaction.guild,
-      channel = interaction.channel as Discord.TextChannel,
-      db = mongoDb,
-      discord = Discord,
-      client = Client
-
+  async execute(interaction: Discord.CommandInteraction) {
```

Comment:
Same as above

Reply:
Totally not what I’m talking about. Don’t change the constants and don’t remove unused declarations on eval, they’re there for a reason. And yes the function does not need to be async

Reply:
Oops, I didn't think about the fact that they would be used inside eval.

Reply:
Still not fixed

Reply:
They should be fixed now.